heal/vlmd/schemas/heal_csv.py

```diff
@@ -0,0 +1,782 @@
+heal_csv_schema = {
```

Comment:
I wonder if we should keep these schemas in JSON format if we want to keep them individually, might be easier to work with?

Reply:
Yes, that is a good idea. The files will need a little re-formatting - they are not valid json right now. Re-formatting can be a big improvement in these two files.